src/ucs/Makefile.am

```diff
@@ -177,6 +177,10 @@ libucs_la_SOURCES = \
 	type/thread_mode.c \
 	vfs/base/vfs_obj.c
 
+if HAVE_CUDA
+libucs_la_SOURCES += \
+        profile/nvtx/nvtx.c
+endif
```

Comment:
space line after

Reply:
removed this after moving range implementation to profile.c